plugins/woocommerce/src/Internal/Email/OrderPriceFormatter.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * OrderPriceFormatter class file.
+ */
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Internal\Email;
+
+use WC_Abstract_Order;
+
+/**
+ * Helper class for formatting prices in order emails.
+ *
+ * @internal Just for internal use.
+ */
+class OrderPriceFormatter {
+
+	/**
+	 * Gets item subtotal - formatted for display in emails.
+	 *
+	 * @param WC_Abstract_Order $order Order instance.
+	 * @param object            $item Item to get unit price from.
+	 * @param string            $tax_display Incl or excl tax display mode.
+	 * @return string Formatted item subtotal.
+	 */
+	public static function get_formatted_item_subtotal( WC_Abstract_Order $order, $item, string $tax_display = '' ): string {
+		$tax_display   = $tax_display ? $tax_display : get_option( 'woocommerce_tax_display_cart' );
```

Comment:
I think I understand why we need this line. but why not remove it and make `$tax_display` mandatory?

We still be able to pass `get_option( 'woocommerce_tax_display_cart' )` from templates, 
and this class gets a bit more isolated.